jdi-light-html/src/main/java/com/epam/jdi/light/ui/html/asserts/TextAreaAssert.java

```diff
@@ -63,4 +63,10 @@ public TextAreaAssert lines(String... lines) {
     public TextAreaAssert lines(List<String> lines) {
         return lines(containsInAnyOrder(lines.toArray()));
     }
+
```

Comment:
Please add code only to folders with name "material ui". See folder jdi-light-material-ui, how there's done for other elements